controllers/space/space_controller_test.go

```diff
@@ -1035,47 +976,47 @@ func TestRetargetSpace(t *testing.T) {
 			HasConditions(spacetest.Retargeting()).
 			HasStatusTargetCluster("member-1") // not reset yet
 
-		t.Run("status target cluster is not reset while NSTemplateSet is still being deleted on member-1", func(t *testing.T) {
-			// once NSTemplateSet resource is fully deleted, the SpaceController is triggered again
-			// and it can create the NSTemplateSet on member-2 cluster
-
+		//t.Run("status target cluster is not reset while NSTemplateSet is still being deleted on member-1", func(t *testing.T) {
```

Comment:
do we need to keep this?

Reply:
I forgot to push a commit that removes it 7dcc5402